project/{% if 'github.com' in source_url %}.github{% endif %}/workflows/pr.yml.j2

```diff
@@ -22,5 +22,5 @@ jobs:
       contents: write
       id-token: write
       pages: write
-      pull-requests: read
+      pull-requests: write
```

Comment:
Why this is necessary?

Reply:
It's necessary for pushes to `main` to be able to submit/update the release PR.